apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -58,19 +59,29 @@ public boolean isValidMessage(FissClaimChange fissClaimChange) {
     return !InvalidDcnRegex.matcher(fissClaimChange.getDcn()).find();
   }
 
-  /** {@inheritDoc} */
+  /**
+   * This implementation checks the {@link FissClaimChange#getChangeType}.
+   *
+   * <p>{@inheritDoc}
+   *
+   * @param fissClaimChange message to check
+   * @return true if {@link FissClaimChange#getChangeType} is {@link ChangeType#CHANGE_TYPE_DELETE}.
+   */
+  @Override
+  public boolean isDeleteMessage(FissClaimChange fissClaimChange) {
+    return fissClaimChange.getChangeType() == ChangeType.CHANGE_TYPE_DELETE;
+  }
+
   @Override
   public String getClaimIdForMessage(FissClaimChange object) {
     return object.getClaim().getRdaClaimKey();
   }
 
-  /** {@inheritDoc} */
```

Comment:
While I didnt add the check to require these in checkstyle (i guess I should have), I added them everywhere just to denote the expected style and indicate the documentation was considered.

Reply:
Comments consisting solely of `{@inheritDoc}` are not considered best practice (in fact they are a warning in the IDEA IDE's javadoc checks) and they add no value since javadoc will include the inherited doc automatically if no javadoc is provided.  The only reason for using `@inheritDoc` is to insert the inherited doc when the method has its own specific comments as well.